Manifests/ManifestsApple/com.apple.mdm.plist

```diff
@@ -374,6 +374,52 @@
 			<key>pfm_type</key>
 			<string>boolean</string>
 		</dict>
+		<dict>
+			<key>pfm_ios_min</key>
+			<string>15</string>
+			<key>pfm_macos_min</key>
+			<string>12</string>
+			<key>pfm_name</key>
+			<string>AssignedManagedAppleID</string>
+			<key>pfm_title</key>
+			<string>Assigned Managed Apple ID</string>
+			<key>pfm_tvos_min</key>
+			<string>15</string>
+			<key>pfm_type</key>
+			<string>string</string>
+		</dict>
+		<dict>
+			<key>pfm_ios_min</key>
```

Comment:
Per documentation, despite only one option being available, the default is `BYOD`, so let's be expicit:
```
<key>pfm_default</key>
<string>BYOD</string>
```

Reply:
I think that this was a documentation goof that got fixed because there's no mention of a default for that key right now and usually the proper default for a key like that is unset (`nil`).